README.md

```diff
@@ -20,9 +20,14 @@ caters to the needs of both confident and less confident cyclists.
 
 ## The Latest Version
 
-- Latest version https://play.google.com/store/apps/details?id=net.cyclestreets
+<a href="https://play.google.com/store/apps/details?id=net.cyclestreets" title="net.cyclestreets"><img src="http://pluspng.com/img-png/get-it-on-google-play-png-get-it-on-google-play-646.png" alt="Get It On Google Play PNG" width ="150" height ="65" ></a>
+
 - Source at http://www.cyclestreets.net/mobile/android/
 
+## Screenshots
+<img src="/imgs/1.png" alt="Screenshots" height = 400>  <img src="/imgs/2.png" alt="Screenshots" height = 400>  <img src="/imgs/3.png" alt="Screenshots" height = 400>  <img src="/imgs/4.png" alt="Screenshots" height = 400>  <img src="/imgs/5.png" alt="Screenshots" height = 400>  <img src="/imgs/6.png" alt="Screenshots" height = 400>
```

Comment:
I presume you obtained these screenshots from https://www.cyclestreets.net/mobile/android/.  Two thoughts:

1. I don't see the point in duplicating the actual images here, rather than referring to the images hosted there.
2. I don't understand why only a subset of images have been shown, and their (meaningful) individual names and alt texts have been reduced to numbers and "Screenshots".

Reply:
Yes, I could have done it. Thanks for suggestions. I will improve this state of files.